### ISSUE TITLE
Unable to load tiny yolo weights

### ISSUE THREAD

**Issue by dAmnation69** [experiencor/keras-yolo2#327]
I have literally tried every source available for tiny yolo weights. But whenever i tried to load them i get the error `Unable to open file (file signature not found)`. 
Can someone explain to me what I am doing wrong or guide me ?

Much appreciated. 

Cheers , Peace

**Comment by SemabAli** [comment]
did you find solution i am also facing this issue

**Comment by dAmnation69** [comment]
Yes, I downloaded the original weights from darknet and converted them using [YAD2K](https://github.com/allanzelener/YAD2K)
